<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterPage</class>
 <widget class="QWidget" name="RegisterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VibeTube - Register Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #FFE6F0;
    color: black; 
}
QPushButton {
    background-color: #FF6F91;
    color: white;
    font-weight: bold;
    border-radius: 8px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #FF9AA2;
}
QLineEdit, QTextEdit, QComboBox {
    background-color: #FFFFFF;
    border: 2px solid #FFC1CC;
    border-radius: 6px;
    padding: 4px;
    color: black; /
}
   
    QLineEdit::placeholder { 
        color: grey; /* Optional: Makes placeholder text grey for contrast */
    }
    </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold; color: black;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameInput">
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="email_line">
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordInput">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="roleSelect">
     <item>
      <property name="text">
       <string>Viewer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Creator</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registerBtn">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>